Minio.Functional.Tests/FunctionalTest.cs

```diff
@@ -2533,8 +2662,6 @@ bool isAWS(string endPoint)
         finally
         {
             await TearDown(minio, bucketName);
-            if (subscription != null)
```

Comment:
Since the build test issues, stem from listen notification tests, are not related to this PR, this and all notification related changes need to be cleaned up here.
I'll be working on it.